runners/samza/src/main/java/org/apache/beam/runners/samza/runtime/SamzaMetricsBundleProgressHandler.java

```diff
@@ -148,7 +151,10 @@ private void parseAndUpdateMetric(MetricsApi.MonitoringInfo monitoringInfo) {
         break;
 
       default:
-        LOG.warn("Unsupported metric type {}", monitoringInfo.getType());
+        if (LOG.isDebugEnabled()) {
```

Comment:
IMO, this check is overkill as the log message itself is not constructing complex statements (string)? I think it is better to just do `LOG.debug`  

Additionally it is just more readable than this additional branching.

Reply:
+1, LOG.debug() checks debug enabled itself.

Reply:
Yes it's overkill.

Reply:
fixed